src/core/public/application/application_service.tsx

```diff
@@ -92,6 +92,7 @@ export class ApplicationService {
   private registrationClosed = false;
   private history?: History<any>;
   private navigate?: (url: string, state: unknown, replace: boolean) => void;
+  private navigateToNewTab?: (url: string) => void;
```

Comment:
NIT: `openInNewTab(url)` is probably more explicit.

Reply:
Updated in [`c1d3f3f` (#96723)](https://github.com/elastic/kibana/pull/96723/commits/c1d3f3f0cf8e8258c02a393daa71c74ca161fc46)